plugins/kubernetes-cluster/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@backstage/plugin-kubernetes-cluster",
+  "description": "A Backstage plugin that integrates towards Kubernetes",
```

Comment:
May want to differentiate this from the plugin-Kubernetes, which has an identical description, esp for folks who may be searching npmjs.com.

Reply:
done